docs/how-to/h-create-and-list-backups.md

```diff
@@ -0,0 +1,14 @@
+Creating and listing backups requires that you:
+* [Have a cluster with at least three nodes deployed](https://discourse.charmhub.io/t/charmed-mysql-tutorial-managing-units/TODO)
+* Access to S3 storage
+* [Have configured settings for S3 storage](https://discourse.charmhub.io/t/configuring-settings-for-s3/TODO)
+
+Once you have a three nodes cluster that has configurations set for S3 storage, check that Charmed MySQL is `active` and `idle` with `juju status`. Once Charmed MySQL is `active` and `idle`, you can create your first backup with the `create-backup` command:
+```
+juju run-action mysql/leader create-backup --wait
```

Comment:
we actually need to run this command on a non-leader unit if more than one node is deployed to create a backup